libcaf_core/caf/actor_system.cpp

```diff
@@ -417,6 +419,8 @@ class actor_system::impl {
       auto mod_id = mod_ptr->id();
       modules[mod_id].reset(mod_ptr);
     }
+    // Let there be daemons.
+    modules[actor_system_module::daemons].reset(new detail::daemons);
```

Comment:
Any reason this is separate from the `module_factories` approach used for the rest?

Reply:
Ah, didn't comment on this one. Basically, we'll always load the daemons module it's not something that we need to be configurable.